manifests/shared/deployments/ipsec.yml

```diff
@@ -0,0 +1,9 @@
+meta:
+  ipsec:
+    targets:
+    - 10.0.32.0/24
```

Comment:
I've made it but I strongly dislike that change. Now we have to synchronise 3 files if we want to add one more subnet for cells. Previously it was two.